concent_api/core/tests/test_unit_validation.py

```diff
@@ -18,7 +20,8 @@
 from core.exceptions import FrameNumberValidationError
 from core.exceptions import HashingAlgorithmError
 from core.exceptions import Http400
-from core.validation import validate_all_messages_identical
+from core.exceptions import MessageValidationError
+from core.validation import validate_all_messages_identical, validate_compute_task_def
```

Comment:
Same here, :+1: for making them alphabetical.

Reply:
done